apparmor.d/abstractions/systemd-common

```diff
@@ -3,8 +3,14 @@
 # Copyright (C) 2021-2024 Alexandre Pujol <alexandre@pujol.io>
 # SPDX-License-Identifier: GPL-2.0-only
 
+  capability sys_ptrace,
+
   ptrace (read) peer=@{systemd},
 
+  owner @{lib}/systemd/{,systemd} r,
+
+  owner @{run}/systemd/system/ r,
```

Comment:
What profile need this?

`@{lib}/systemd/{,systemd} r,` should already be covered by the base abstraction.